docs/reference/query.rst

```diff
@@ -188,6 +188,33 @@ Find all items with a file modification time between 2008-12-01 and
 
     $ beet ls 'mtime:2008-12-01..2008-12-02'
 
+You can also add an optional time value to date queries, specifying hours,
+minutes, and seconds.
+
+Times are separated from dates by a space, an uppercase 'T' or a lowercase
+'t', for example: ``2008-12-01T23:59:59``. If you specify a time, then the
+date must contain a year, month, and day. The minutes and seconds are
+optional.
+
+Here is an example that finds all items added on 2008-12-01 at or after 22:00
+but before 23:00::
+
+    $ beet ls 'added:2008-12-01T22'
```

Comment:
I assume you're asking why there isn't a `..` in the query, followed by the right hand part of the range?

This is because by specifying the hour of 22, results from anywhere within that hour are returned. The right hand part of the range is implied.

This is the same way that date queries already work.